experiments/spotify/2_consume_source/spotify_pipeline.py

```diff
@@ -0,0 +1,18 @@
+from dlt import pipeline as p
+from dlt.sources import spotify
+
+# you can configure your pipeline explicitely
+# if not configured, sensible defaults will be used: the current file name will become the name of the pipeline
+# the working dir goes to tmp and is recreated each time pipeline is run
+# this also writes `.dlt/config` file
+# dlt.config("spotify_pipeline", work_dir="~/pipelines/spotify")
+
+# this get a list of iterators with associated schema
+data = spotify.get_data("/path/to/data", tables=["songs", "history"])
```

Comment:
this is another good question. how do we name methods that return data. are this name always the same? can a source have several alternative functions that return data? in case of spotidy you could have

```
@source()
def spotify_file(data_dir, tables): ...

@source()
def spotify_api(user_id, credentials, tables): ...

```